gh-actions/go/gotestfmt/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Install gotestfmt and our wrapper script
+description: Install gotestfmt and our wrapper script which improves the output and copies it to log files
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install gotestfmt
+      run: go install github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@latest
```

Comment:
Should we pin to a specific version to ensure that action users can control when this is updated?

Or can instead add an action input and control the version from it (defaulting to latest if you want in that case).

Reply:
Good idea to control it via an action input. I'll do that.

Reply:
Done and tested in https://github.com/ubuntu/authd-oidc-brokers/actions/runs/11820452458/job/32935582415